tests/codeception/frontend/functional/SignupCest.php

```diff
@@ -10,29 +10,27 @@ class SignupCest
 
     /**
      * This method is called before each cest class test method
-     * @param \Codeception\Event\TestEvent $event
+     * @param \codeception_frontend\FunctionalTester $I
      */
-    public function _before($event)
+    public function _before($I)
```

Comment:
[Pretty shure](https://github.com/Codeception/Codeception/blob/2.0/src/Codeception/TestCase/Cest.php#L207) It can be an event in a module, but the codeception doc [tells nothing about events in module hooks](https://github.com/Codeception/Codeception/blob/2.0/docs/03-ModulesAndHelpers.md#hooks).